SU2_CFD/include/fluid/CFluidModel.hpp

```diff
@@ -137,6 +139,40 @@ class CFluidModel {
    */
   su2double GetCv() const { return Cv; }
 
+  /*!
+   * \brief flamelet LUT - Get the source term of the transported scalar
+   */
+  virtual inline const su2double* GetScalarSources() const { return nullptr; }
```

Comment:
Use only the pointer version for simplicity of the interface please

Reply:
OK, so we remove the other value-based function `su2double GetScalarSources() `?